packages/documentation/docs/controls/application-frame/_application-header_styleguide.md

```diff
@@ -1,4 +1,4 @@
-import Playground from '@site/src/components/PlaygroundV2';
+import Playground from '@site/src/components/PlaygroundV3';
```

Comment:
Why do we need the Playground inside the styleguide in general? It seems not to be used